src/lua/lua_script.rs

```diff
@@ -84,6 +84,10 @@ fn create_default_lua_state(
             .load(include_str!("../../resources/lua/tasks.lua"))
             .call::<_, rlua::Value>(())?;
         globals.set("tasks", lua_tasks)?;
+        let lua_triggers = ctx
+            .load(include_str!("../../resources/lua/triggers.lua"))
+            .call::<_, rlua::Value>(())?;
+        globals.set("triggers", lua_triggers)?;
```

Comment:
Id prefer if the module is named `trigger`. For me it feels nicer with `trigger.add()` and `trigger.remove()` etc.

Reply:
Hmm, in that case we should probably rename `tasks` to `task`, as well

Reply:
On an additional note. We don't need to require this one through rust do we? We could just require it in one of the lua files. Eg. Add one called systems which loads them?

Reply:
Or perhaps we do to make sure they get baked into memory...

Reply:
Yeah, currently they are part of the executable.
The minecraft mods that run Lua code tend to not place their libraries in `_G` and instead require you to always `require` them. I'm not sure we can do that easily in Blightmud without going away from the single executable, which I honestly quite like.

It's certainly possible - worst case, we monkey-patch `require` - but that seems a bit... drastic.

Reply:
Nah. We're sticking with single executable. We will do it like this for our mods and use the other way for plugins.